test/auth/oidc_test.go

```diff
@@ -95,3 +97,23 @@ func TestParallelSupportsOIDC(t *testing.T) {
 
 	env.Test(ctx, t, oidc.ParallelHasAudienceOfInputChannel(name, env.Namespace(), channel_impl.GVR(), channel_impl.GVK().Kind))
 }
+
+func TestSequenceSupportsOIDC(t *testing.T) {
+	t.Parallel()
+
+	ctx, env := global.Environment(
+		knative.WithKnativeNamespace(system.Namespace()),
+		knative.WithLoggingConfig,
+		knative.WithTracingConfig,
+		k8s.WithEventListener,
+		environment.Managed(t),
+	)
+
+	name := feature.MakeRandomK8sName("sequence")
+	env.Prerequisite(ctx, t, sequencefeatures.GoesReady(name, sequence.WithChannelTemplate(channel_template.ChannelTemplate{
+		TypeMeta: channel_impl.TypeMeta(),
+		Spec:     map[string]interface{}{},
+	})))
+
+	env.Test(ctx, t, oidc.SequenceHasAudienceOfInputChannel(sequence.GVR(), sequence.GVK().Kind, name, env.Namespace()))
```

Comment:
```suggestion
	env.Test(ctx, t, oidc.SequenceHasAudienceOfInputChannel(name, env.Namespace(), channel_impl.GVR(), channel_impl.GVK().Kind))
```

`SequenceHasAudienceOfInputChannel` has different parameters: https://github.com/knative/eventing/blob/17572031cad40b45c267d543fd56ba6ac8904a69/test/auth/features/oidc/sequence.go#L29

Should fix the build issues:
```
# knative.dev/eventing/test/auth [knative.dev/eventing/test/auth.test]
test/auth/oidc_test.go:118:58: cannot use sequence.GVR() (value of type "k8s.io/apimachinery/pkg/runtime/schema".GroupVersionResource) as string value in argument to oidc.SequenceHasAudienceOfInputChannel
test/auth/oidc_test.go:118:95: cannot use name (variable of type string) as "k8s.io/apimachinery/pkg/runtime/schema".GroupVersionResource value in argument to oidc.SequenceHasAudienceOfInputChannel
```